GVFS/GVFS.Service/GVFSService.Mac.cs

```diff
@@ -20,18 +20,18 @@ public class GVFSService
         private IRepoRegistry repoRegistry;
         private RequestHandler requestHandler;
 
-        public GVFSService(
-            ITracer tracer,
-            string serviceName,
-            IRepoRegistry repoRegistry)
+        public GVFSService(
+            ITracer tracer,
+            string serviceName,
+            IRepoRegistry repoRegistry)

```

Comment:
Are these changes caused by File Encoding differences between Windows and Mac? This has become a common issue while editing cross-plat files.

Reply:
yes... I have wondered if we could help solve this with a setting in editorconfig. I think the expectation is that all lines are crlf terminated, but that gets mixed up...

Reply:
Setting up the `.editorconfig` to enforce line endings and normalizing the files that are currently mixed has been on my backlog of things I wanted to do for a long time. I think the difficult part there is validating that it behaves as expected in all the IDEs we use.